src/universal-dapp.js

```diff
@@ -826,8 +827,11 @@ UniversalDApp.prototype.rawRunTx = function (args, cb) {
         ++self.blockNumber
       }
       self.vm.runTx({block: block, tx: tx, skipBalance: true, skipNonce: true}, function (err, result) {
-        result.transactionHash = self.txdebugger.web3().releaseCurrentHash() // used to keep track of the transaction
```

Comment:
I sitll don't get where is `result` readonly? Safari just takes any input parameter as readonly?